src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/monitor-services/tutorial-monitor-redis-running-kubernetes.mdx

```diff
@@ -36,43 +37,39 @@ If you'd like to first complete the [Kubernetes tutorial](https://kubernetes.io/
 
 The PHP Guestbook application has three Redis instances: one master and two slave instances. Each instance is tagged with a label where `app=redis`. For this example, we're using our [Redis monitoring integration](/docs/integrations/host-integrations/host-integrations-list/redis-monitoring-integration). It can monitor both master and slave instances of Redis, so we don’t have to distinguish between them.
 
-1. In the Kubernetes integration's YAML config file (`newrelic-infrastructure-k8s-latest.yaml`), you need to update the `nri-integration-cfg` section. From [the list of integration configs](/docs/monitor-service-running-kubernetes#integration-configs), get the Redis integration YAML and add it to the Kubernetes config. The Redis YAML is highlighted below.
+1. In the `values-newrelic.yaml` YAML file for our [helm chart](/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm) you need to update the `newrelic-infrastructure.integrations` section. From [the list of integration configs](/docs/monitor-service-running-kubernetes#integration-configs), get the Redis integration YAML and add it to the Kubernetes config. The Redis YAML is highlighted below.
```

Comment:
no V2 info anymore here right? Should we add a warning till it is beta?

Reply:
adding a callout/warning maybe?

Reply:
Added in https://github.com/newrelic/docs-website/pull/5742/commits/6d66f78075c4c1ebfa3b2ac8b8082086bcebafea